### PR TITLE
Boxicons

### LINKED ISSUE #539: [Add] Boxicons
### Site URL

https://boxicons.com/

### Description of the above provided site

Boxicons is a versatile collection of over 2,000 elegant vector icons, enhancing websites, apps, and digital projects. With clean lines and sharp edges, these icons seamlessly integrate with diverse design styles. Customize and bring your creative vision to life with Boxicons.

### Category

Frontend

### Sub Category

icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #539

## Changes proposed

Boxicons: 2,000+ elegant vector icons for websites, apps, and digital projects. Clean lines, easy customization.

## Screenshots

Screenshot (799)

1. Updated description
   
Screenshot (801)

## Note to reviewers

Any kind of feedback would be appreciated

### CODE REVIEW

**Comment by parma1111** [comment]
@rupali-codes @CBID2

File: database/frontend/icons.json [hunk 1]
```diff
@@ -104,6 +104,13 @@
     "category": "frontend",
     "subcategory": "icons"
   },
+  {
+    "name": "Boxicons",
+    "description": "Boxicons offer 2,000+ elegant vector icons for websites, apps, and digital projects. Clean lines, easy customization.",
```
Comment (CBID2) [inline database/frontend/icons.json:109]:
```suggestion
    "description": "Boxicons offer 2,000+ elegant vector icons for websites, apps, and digital projects. Clean lines and easy customization.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @parma1111! :) You're off to a great start! :) I just made a little change to your description. :)

**Review by ujjawaltyagii — APPROVED** [review]
Lgtm !!

**Review by parma1111 — COMMENTED** [review]
done

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by CBID2 — APPROVED** [review]
LGTM!

**Pull request merged**